client/src/main/java/io/pravega/client/stream/impl/ReaderGroupState.java

```diff
@@ -211,6 +211,7 @@ boolean isReaderOnline(String reader) {
 
     /**
      * Returns the number of segments currently being read from and that are unassigned within the reader group.
+     * @return The number of segments currently being read from and that are unassigned within the reader group.
```

Comment:
These Javadoc changes are unrelated to the change at hand. Please undo them and submit them using a separate PR.